src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -17,16 +18,25 @@ class PdlClient(
     private val azureAdV2Client: AzureAdV2Client,
     private val pdlClientId: String,
     private val pdlUrl: String,
+    private val redisStore: RedisStore,
 ) {
     private val httpClient = httpClientDefault()
 
     suspend fun navn(
         personIdent: PersonIdent,
     ): String {
-        val token = azureAdV2Client.getSystemToken(pdlClientId)
-            ?: throw RuntimeException("Failed to send request to PDL: No token was found")
-        return person(personIdent, token)?.fullName()
-            ?: throw RuntimeException("PDL returned empty navn for given fnr")
+        val cacheKey = "$NAVN_CACHE_KEY_PREFIX${personIdent.value}"
+        val cachedNavn: String? = redisStore.get(key = cacheKey)
+        return if (cachedNavn != null) {
+            cachedNavn
+        } else {
+            val token = azureAdV2Client.getSystemToken(pdlClientId)
+                ?: throw RuntimeException("Failed to send request to PDL: No token was found")
+            val navn = person(personIdent, token)?.fullName()
```

Comment:
Jeg ser at denne burde oppdateres, slik som vi har i `syfooversiktsrv` for å støtte flere fornavn og bindestrek. https://github.com/navikt/syfooversiktsrv/pull/204/files

Reply:
Ja, antakelig lurt å fikse det i en egen PR. Jeg bare la på caching her siden det ble mulig.